cmd/system-probe/modules/gpu_monitoring.go

```diff
@@ -12,6 +12,7 @@ import (
 	"net/http"
```

Comment:
can we rename that simply go `gpu.go` ? it suits better go naming conventions

Reply:
Renamed